frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/provision-source.tsx

```diff
@@ -64,11 +70,22 @@ const ProvisionSourceNetHelpMsg: React.FC<ProvisionSourceNetHelpMsgProps> = ({
   );
 };
 
-export const ProvisionSourceComponent: React.FC<ProvisionSourceComponentProps> = React.memo(
-  ({ provisionSourceField, onChange, goToStorageStep, goToNetworkingStep }) => {
+export const ProvisionSourceComponentWithNamespace: React.FC<ProvisionSourceComponentProps> = React.memo(
+  ({ provisionSourceField, onChange, goToStorageStep, goToNetworkingStep, activeNamespace }) => {
     const provisionSourceValue = iGetFieldValue(provisionSourceField);
     const sources = iGet(provisionSourceField, 'sources');
 
+    const networkAttachmentDefinitionModelResource = {
```

Comment:
let's fetch this wizard wide. We should also double check the permissions, so that the wizard is usable for non-admin users

Reply:
fetching on create wizard level